site/content/contribute/more-info/server/schema-migration-guide.md

```diff
@@ -44,16 +339,9 @@ The problem arises when in some databases, for some tables, due to various techn
     - If it's applied you can revert it using morph CLI command. An example command would look like `morph apply down --driver {your-driver} --dsn "{your-dsn}" --path {path-to-your-driver-specific-migration-files} --number 1`
 2. If the migration has been shipped in a release and you want to apply fixes, instead of changing the existing script, you should add a new one so that `db_migrations` will stay consistent. You can edit the existing migration to be a no-op for future releases in this case.
 
-### How do I measure the impact of the migration?
-
-Answer the following two questions:
-1. Is it a large table (like `Posts` or `ThreadMemberships`)? (The performance team can help you with this.)
-2. Can my change run concurrently with other database operations in that table? Check MySQL and PostgreSQL documentation, starting from required minimum versions supported by Mattermost.
-
-If the answer to 2. is yes, then it should be pretty simple. In that case, if the answer to 1 is also yes, then please extract the raw SQL query to be run and add it in the release notes for customers who want to run it in off-hours to speed up upgrade time.
-
-However, If the answer to the second question above is "no", then contact the {{< newtabref href="https://community.mattermost.com/core/channels/developers-performance" title="performance team" >}} to discuss a solution.
+## GLOSSARY
 
-### Inform SRE team
```

Comment:
I think we should still keep this recommendation, just in cae :)

Reply:
This was initially introduced by the SRE team but is not enforced anymore at the moment. And moreover, we would be stricly following non-blocking migrations and along with Community moving to Postgres, I don't think this is necessary anymore. LMK.

Reply:
Fair enough, let's remove this section.